requirements.txt

```diff
@@ -1,3 +1,91 @@
-typer
```

Comment:
@putmantime I did a `pip freeze > requirements.txt` to allow for consistency. Otherwise, if I install from this:

```
typer
linkml-owl
pandas
```

it might not be stable if a breaking change or conflict happens in later versions of these in the future.

The GitHub actions use this fixed `requirements.txt` for pip installation.